src/pem-label.enum.ts

```diff
@@ -0,0 +1,7 @@
+export const PemLabel = {
```

Comment:
What was the reasoning behind putting this type in its own file instead of with all the other types?